src/main/java/com/eternalcode/reports/command/ReportCommand.java

```diff
@@ -0,0 +1,73 @@
+package com.eternalcode.reports.command;
+
+import com.eternalcode.reports.configuration.ConfigurationManager;
+import com.eternalcode.reports.configuration.PluginConfiguration;
+import com.eternalcode.reports.data.Statistics;
+import com.eternalcode.reports.message.GlobalMessages;
+import com.eternalcode.reports.util.DiscordWebHookCreator;
+import com.eternalcode.reports.util.NotificationManager;
+import dev.rollczi.litecommands.argument.Arg;
+import dev.rollczi.litecommands.argument.Name;
+import dev.rollczi.litecommands.argument.joiner.Joiner;
+import dev.rollczi.litecommands.command.execute.Execute;
+import dev.rollczi.litecommands.command.route.Route;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+
+@Route(name = "report", aliases = {"zglos"})
+public class ReportCommand {
+    private Statistics statistics;
+
+    private GlobalMessages messages;
+    private NotificationManager notificationManager;
+
+    private DiscordWebHookCreator discordWebHookCreator;
+
+    private PluginConfiguration pluginConfiguration;
+    private ConfigurationManager configurationManager;
+
+    public ReportCommand(
+        Statistics statistics,
+        GlobalMessages messages,
+        NotificationManager notificationManager,
+        PluginConfiguration pluginConfiguration,
+        ConfigurationManager configurationManager
+    ) {
+        this.statistics = statistics;
+        this.messages = messages;
+        this.discordWebHookCreator = new DiscordWebHookCreator(pluginConfiguration);
+        this.notificationManager = notificationManager;
+        this.pluginConfiguration = pluginConfiguration;
+        this.configurationManager = configurationManager;
+    }
+
+    @Execute(min = 2)
+    public void reportPlayer(Player sender, @Arg @Name("target") Player target, @Joiner @Name("message") String message) {
+        if (sender.getUniqueId() == target.getUniqueId()) {
+            this.notificationManager.announceMessage(sender.getUniqueId(), this.messages.userMessages.canNotReportYourself);
+            return;
+        }
+
+        Bukkit.getOnlinePlayers()
+            .stream()
+            .filter(player -> player.hasPermission("eternalcode.report.recieve"))
+            .forEach(player -> {
+                this.notificationManager.announceMessage(
+                    player.getUniqueId(),
+                    this.messages.userMessages.reportForAdministrator
+                        .replace("{USER}", target.getName())
+                        .replace("{REASON}", message)
+                        .replace("{REPORTED_BY}", sender.getName())
+                );
+            });
```

Comment:
Dodaj "Server" do konstruktora i 👇

```suggestion
        this.server.getOnlinePlayers()
            .stream()
            .filter(player -> player.hasPermission("eternalcode.report.recieve"))
            .forEach(player -> {
                this.notificationManager.announceMessage(
                    player.getUniqueId(),
                    this.messages.userMessages.reportForAdministrator
                        .replace("{USER}", target.getName())
                        .replace("{REASON}", message)
                        .replace("{REPORTED_BY}", sender.getName())
                );
            });
```